iptables/init.sls

```diff
@@ -1,13 +1,14 @@
 # Firewall management module
+{% set packages = salt['grains.filter_by']({
+    'Debian': ['iptables', 'iptables-persistent'],
+    'RedHat': ['iptables'],
+    'default': 'Debian'}) %}
+
 {%- if salt['pillar.get']('firewall:enabled') %}
```

Comment:
How about you set a default for this so in case this is not present in the pillar, by default the firewall should be enabled.